src/fragments/lib/analytics/flutter/getting-started/20_installLib.mdx

```diff
@@ -1,18 +1,18 @@
-In your Flutter project directory, open **pubspec.yaml**.  
+In your Flutter project directory, open **pubspec.yaml**.
 
-> You will already have configured Amplify by following the steps in the project setup. 
+> You will already have configured Amplify by following the steps in the project setup.
 
-Add Analytics by adding these libraries into your dependencies block: 
+Add Analytics by adding these libraries into your dependencies block:
 
-```yaml 
+```yaml
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.17.3 <3.0.0"
 
 dependencies:
 
-  # Should already be added during Project Setup walkthrough 
-  amplify_flutter: ^0.5.0
+  # Should already be added during Project Setup walkthrough
+  amplify_flutter: ^0.5.1
```

Comment:
I agree - we should just keep this to 0.6.0 for our projects. @salihgueler can we update this everywhere so that with semantic versioning developers are getting our latest and greatest?

Reply:
This is already updated. This code is marked as outdated.